### ISSUE TITLE
Functions that are commands should include bindings if bound

### ISSUE THREAD

**Issue by hmelman** [minad/marginalia#21]
I wish that candidates that are functions (like with `describe-function`) that are in fact bound commands would display their keybindings the way they do for commands (as in `M-x`).  Not only is this useful info, it makes bound commands jump out in a list of functions.

**Comment by oantolin** [comment]
That's easy, just configure your Marginalia as follows:

```elisp
(setf (alist-get 'describe-function marginalia-command-categories) 'command)
```

**Comment by minad** [comment]
@oantolin There is still the question if we should somehow do this automatically for symbols and restructure how we do things right now. We could replace marginalia-annotate-command-full by marginalia-annotate-symbol. And marginalia-annotate-symbol could call marginalia-annotate-command-binding if the symbol is a binding. I thought about this before.

**Comment by oantolin** [comment]
I have no preference. I don't particularly want key bindings by default in `C-h f` but I wouldn't mind at all if they were present either.

**Comment by minad** [comment]
@oantolin I implemented this in https://github.com/minad/marginalia/commit/f16d26a17bb93684dd0f099db6dc0400a743dcc3. Since we prefer the function description for symbols over the variable description, I think it makes sense to also show bindings.

**Comment by hmelman** [comment]
Thanks, using `marginalia-command-categories` works.  It wasn't immediately obvious to me (I had looked) and it might make a good entry in the README  how to do this.  I like doing this by default for two reasons.  Once I had this I used it all the time for exploring things.  I would lookup functions based on "topics" like "sentence" or "word" and see both functions and commands with a 1 line doc and quickly find options both for using commands and writing functions.  Also I use the package `helpful` rather than the describe- commands and if it automatically got this capability that would be great.
